pkg/dockerutil/dockerutils.go

```diff
@@ -412,6 +412,12 @@ func GetDockerIP() (string, error) {
 // Example code from https://gist.github.com/fsouza/b0bf3043827f8e39c4589e88cec067d8
 func RunSimpleContainer(image string, name string, cmd []string, entrypoint []string, env []string, binds []string, uid string) (string, error) {
 	client := GetDockerClient()
+	var buf bytes.Buffer
+	err := client.PullImage(docker.PullImageOptions{Repository: image, OutputStream: &buf},
```

Comment:
I suspect there's a more elegant way to check to see if we have the container before pulling it, but think this is good enough for now.